### PR TITLE
updating fixes

### PR DESCRIPTION
**Pull request by iruzevic**
- adding textdomain correct replace
- fixing some warnings in search/replace
- fixing @package rename

### CODE REVIEW

File: src/Cli/AbstractCli.php [hunk 1]
```diff
@@ -465,6 +479,22 @@ public function renameUseFrontendLibs(array $args = []): self
 		return $this;
 	}
 
+	/**
+	 * Convert camel case to kebab case.
+	 *
+	 * @param string $input Input to convert.
+	 * @return string
+	 */
+	public function camelCaseToKebabCase(string $input): string
```
Comment (IvanGrginovInf) [inline src/Cli/AbstractCli.php:488]: Unit test for this one por favor 🙏

Comment (iruzevic) [reply src/Cli/AbstractCli.php:488]: this is not covered fully in the tests so I will add the tests in the next PR.

Setup is failing now so this is a hotfix

File: src/Cli/AbstractCli.php [hunk 2]
```diff
@@ -465,6 +479,22 @@ public function renameUseFrontendLibs(array $args = []): self
...
+	{
+		preg_match_all('!([A-Z][A-Z0-9]*(?=$|[A-Z][a-z0-9])|[A-Za-z][a-z0-9]+)!', $input, $matches);
```
Comment (dingo-d) [inline src/Cli/AbstractCli.php:490]: What's the use of `!` in the regex here exactly?

Seems to me this helper should either be `private` or put in the helpers as a static helper. And tested ofc.

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left one comment. 

Can you add a test for text-domain replacement as well? So that we verify it works correctly.

**Review by IvanGrginovInf — APPROVED** [review]

**Comment by IvanGrginovInf** [comment]
@iruzevic approved but phpstan is failing.

**Comment by iruzevic** [comment]
@IvanGrginovInf fixed phpstan

**Review by IvanGrginovInf — APPROVED** [review]

File: src/Cli/CliHelpers.php [hunk 3]
```diff
@@ -37,4 +37,19 @@ public static function cliError(string $errorMessage): void
...
+	 * @param string $input Input to convert.
+	 * @return string
+	 */
+	public static function camelCaseToKebabCase(string $input): string
+	{
+		$output = ltrim(strtolower((string)preg_replace('/[A-Z]([A-Z](?![a-z]))*/', '-$0', $input)), '-');
+		$output = str_replace('_', '-', $output);
+		$output = str_replace('--', '-', $output);
+
+		return $output;
```
Comment (dingo-d) [inline src/Cli/CliHelpers.php:53]:
```suggestion
		return str_replace('--', '-', $output);
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
One small comment left

**Review by dingo-d — APPROVED** [review]

**Pull request merged**